tests/EdFi.Ods.Api.IntegrationTests/AuthorizationViewHelper.cs

```diff
@@ -7,36 +7,69 @@
 using System;
 using System.Collections.Generic;
 using System.Data;
+using System.Linq;
+
+
 namespace EdFi.Ods.Api.IntegrationTests
 {
     public static class AuthorizationViewHelper
     {
 
-        private static bool QueryAuthorizationByViewName(IDbConnection connection, string  viewName)
+        public static void ShouldNotContainDuplicate(IDbConnection connection,
+            string viewName, PersonType personType,
+            int target, int countValue, params (int, int)[] expectedTuples)
         {
-            string subSql = "";
-            if(viewName.Equals("StudentUSIToEducationOrganizationId",StringComparison.OrdinalIgnoreCase))
-            {
-                subSql= " GROUP BY SourceEducationOrganizationId,studentUSI  HAVING COUNT(*)> 1;";
-            }
-            else if (viewName.Equals("ParentUSIToEducationOrganizationId", StringComparison.OrdinalIgnoreCase))
-            {
-                subSql = " GROUP BY SourceEducationOrganizationId,ParentUSI  HAVING COUNT(*)> 1;";
-            }
+            var actualTuples = GetExistingRecordsInAuthorizationView(connection, viewName, personType, target);
+
+            actualTuples.Count().ShouldBe(countValue);
+            expectedTuples.ShouldBeSubsetOf(actualTuples);
+        }
+
+        public static void ShouldContainTuples(IDbConnection connection, string viewName,
+            PersonType personType, params (int, int)[] expectedTuples)
```

Comment:
If you're going to introduce PersonType, there's no need to pass the view name around.  Just have a small lookup here in this helper that translates PersonType -> View name.

Reply:
I used persontype now and removed viewname parameter